sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -177,6 +179,7 @@ public boolean deactivateUserByLogin(DbSession dbSession, String login) {
     mapper.deleteUserMeasureFilterFavourites(dto.getId());
     mapper.deleteUserProperties(dto.getId());
     mapper.deleteUserRoles(dto.getId());
+    mapper.deletePropertiesMatchingLogin(asList(DEFAULT_ISSUE_ASSIGNEE), dto.getLogin());
```

Comment:
It's a good point but for the moment I have no real opinion on the subject. I propose to discuss about that later.